### PR TITLE
Storedog 2.0

### PR DESCRIPTION
**Pull request by stevedatadog**
See the lab [PR](https://github.com/DataDog/learning-center/pull/166) for notes.

### CODE REVIEW

File: general/storedog-2-0/status.conf [hunk 1]
```diff
@@ -0,0 +1,24 @@
+server {
+   listen 81;
+   server_name localhost;
+
+   access_log off;
+   allow 172.18.0.0/24;
```
Comment (stanleegoodspeed) [inline general/storedog-2-0/status.conf:6]: If we're removing the custom compose network with the specified subnet CIDR, then we'll need to open this up as well

File: general/storedog-2-0/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+    volumes:
+      - /root/postgres:/var/lib/postgresql/data
+      - /root/dd_agent.sql:/docker-entrypoint-initdb.d/dd_agent.sql
+      - /root/restore.sql:/docker-entrypoint-initdb.d/restore.sql
+  redis:
+    image: redis:6.2-alpine
+    volumes:
+      - '/root/redis:/data'
+  puppeteer:
+    image: buildkite/puppeteer:10.0.0
```
[the remark this answers is not in the document]
Comment (devindford) [reply general/storedog-2-0/docker-compose.yml:188]: Heres a link to the branch with the dockerfile I built that works with M1 macs and has the firefox browser dependency we need for it to run on M1s
https://github.com/DataDog/storedog-backend/blob/devin.ford/add_puppeteer/services/puppeteer/Dockerfile

[the remark this answers is not in the document]
Comment (stevedatadog) [reply general/storedog-2-0/docker-compose.yml:188]: This is useful info for development! Here, I'm going to stick with the current image, as it doesn't need to be built and it runs on our VMs.

**Review by arosenkranz — APPROVED** [review]
Looks good! I'm guessing we'll just create new versions of the puppeteer files?

**Pull request merged**